### PR TITLE
added flutter completions

### PR DESCRIPTION
**Pull request by tbhaxor**

### CODE REVIEW

File: completion/available/flutter.completion.bash [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+if command -v flutter > /dev/null; then
```
Comment (nwinkler) [inline completion/available/flutter.completion.bash]: Please use the `_command_exists` function here.

File: completion/available/flutter.completion.bash [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    eval "$(flutter bash-completion)"
```
Comment (nwinkler) [inline completion/available/flutter.completion.bash]: This should be indented with two spaces - please take a look at the `.editorconfig` file in the project root.

File: completion/available/flutter.completion.bash [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+fi
```
Comment (nwinkler) [inline completion/available/flutter.completion.bash]: Please add a line break after the last line.

Comment (tbhaxor) [reply completion/available/flutter.completion.bash]: I am sorry sir its my IDE :man_facepalming:

File: completion/available/flutter.completion.bash (hunk 2 above)
Comment (tbhaxor) [reply completion/available/flutter.completion.bash]: I am sorry sir its my IDE :man_facepalming:

File: completion/available/flutter.completion.bash (hunk 3 above)
Comment (nwinkler) [reply completion/available/flutter.completion.bash]: Please check if your IDE has a plugin for [Editorconfig](https://editorconfig.org) - once you install this, it should automatically do these things (like an extra empty line at the end of the file).

**Comment by nwinkler** [comment]
Thanks!

**Pull request merged**